src/Graph.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <set>
+
+namespace SumColouring
+{
+    class Graph
+    {
+    public:
+        typedef std::pair<int, int> Edge;
+
+        Graph(int vertices, std::set<Edge> edges) : v(vertices), e(edges) {}
```

Comment:
Dawno nie pisałem w C++, ale takich większych argumentów nie przekazywało się przez jakąś referencję albo wskaźnik?

Reply:
w tym przypadku nie jest to konieczne, ale zaraz zmienię żeby nie kopiować niepotrzebnie